package.json

```diff
@@ -45,5 +45,12 @@
   "private": true,
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
+    "handlebars": "^4.3.0",
```

Comment:
since we are resolving these specifically, should we pin them to an exact version? What is the plan going forward to remove the pins? Do I have to remove each pin and run yarn audit?

Reply:
We could pin them to an exact version this basically is saying that sub dependencies need to install at least the version or higher. If we pin them we will lose out on any updates to versions dependencies rely on so might not be the best choice.

>  Do I have to remove each pin and run yarn audit?

You dont have to do anything except run `yarn && yarn audit`

Reply:
so we pin the dependencies to the exact version to have somewhat reproducible builds. I would prefer that we do the same for the resolutions. Ideally, we want to remove these resolutions when the primary dependency updates. I think yarn audit will always report ok when I run it with the resolutions.

Reply:
> I would prefer that we do the same for the resolutions. 

This is not really the right way to do this... We are losing out on all future dependency updates by pinning exact resolutions but I will go ahead and do it anyway